AdminServer/appscale/admin/__init__.py

```diff
@@ -710,6 +880,10 @@ def main():
   app = web.Application([
     ('/v1/apps/([a-z0-9-]+)/services/([a-z0-9-]+)/versions', VersionsHandler,
      all_resources),
+    ('/v1/apps/([a-z0-9-]+)', ProjectHandler,
```

Comment:
We may want to make this a Cloud Resource Manager API call for GCP compatibility (https://cloud.google.com/resource-manager/reference/rest/v1/projects/delete) since the Admin API doesn't support deleting projects.

Reply:
So we'd change '/v1/apps/' to '/v1/projects'? Or are you saying we'd move it completely

Reply:
It probably doesn't need to be moved at this time.

The other major difference is that the method does not return a response body. Instead of polling an operation ID, my guess is that it would be easiest to check if the project ID is still listed in a call to https://cloud.google.com/resource-manager/reference/rest/v1/projects/list.